third-party-podspecs/RNReanimated.podspec.json

```diff
@@ -60,7 +60,6 @@
     "Yoga": [],
     "DoubleConversion": [],
     "glog": [],
-    "React-jsc": [],
```

Comment:
In theory, we no longer use `JSC` as for both platforms we use `hermes`. This dependency was introduced [here when RN version was upgraded to 0.71.11](https://github.com/wordpress-mobile/gutenberg-mobile/commit/0b280a793b4445d67261ff6ab7ae6ac36decdb25#diff-131cf457bc0da81a25ab4a9e90fcd0cfc53feb379cb7f33be34e202471b4766eR63), but seems it wasn't necessary. That said, I wonder if we could remove the podspec (i.e. `third-party-podspecs/React-jsc.podspec.json`).

Reply:
> That said, I wonder if we could remove the podspec (i.e. third-party-podspecs/React-jsc.podspec.json).

I think we could remove it since we are not using it, should we do it with these changes or in a separate PR?

Reply:
Yeah, I agree. I'd rather remove the file in a separate PR as the change shouldn't be related to the RN upgrade.

Reply:
>  I'd rather remove the file in a separate PR as the change shouldn't be related to the RN upgrade.

Sounds good!